src/components/ui/DatePicker/DatePicker.svelte

```diff
@@ -192,12 +194,16 @@
 
   function onChangeViewMonth(event: Event) {
     const { valueAsNumber } = getTarget(event);
-    viewMonth = valueAsNumber;
+    if (valueAsNumber != null) {
```

Comment:
`!=` vs `!==`? Though maybe this was intentional?

Reply:
It's my being lazy haha. If you just do `!= null` it checks against both `null` and `undefined`. I could do a strict check against both if you feel strongly enough about it.

Reply:
Innnteresting, I never knew that!